src/components/LoadingIndicator/LoadingIndicator.jsx

```diff
@@ -79,7 +89,7 @@ const LoadingIndicator = createClass({
 					transitionLeaveTimeout={300}
 				>
 					{isLoading && (
-						<div className={cx('&-message-container', { '&-has-overlay': hasOverlay })}>
+						<div className={cx('&-message-container', { [`&-has-overlay-${overlayKind}`]: hasOverlay })}>
```

Comment:
This could be a breaking change if anyone is selecting the overlay `div` by just the `has-overlay` container for styling, but I assumed that probably wasn't the case.